gulpfile.js

```diff
@@ -40,11 +40,13 @@ function processCss(inputStream, taskType) {
         }))
         .pipe($.newer(paths.styles.dest))
         .pipe($.less({ paths: [$.path.join(__dirname, 'less', 'includes')] }))
-        .pipe($.rename({suffix: '.min'}))
+        .pipe($.autoprefixer({ browsers: ['last 2 versions', '> 5%'] }))
+        .pipe(gulp.dest(paths.styles.dest))
         .pipe($.minifyCss({advanced: false}))
+        .pipe($.rename({ suffix: '.min' }))
         .pipe(gulp.dest(paths.styles.dest))
-        .pipe(browserSync.stream())
-        .pipe($.notify({ message: taskType + ' task complete' }));
+        .pipe(browserSync.stream())/*
+        .pipe($.notify({ message: taskType + ' task complete' }))*/;
```

Comment:
I think these should be uncommented in the actual repo and you can then comment them out locally, as they provide a good indicator of things taking place so you can see a bit more clearly where thing's are going wrong.